docs/source/library/Run.rst

```diff
@@ -0,0 +1,6 @@
+run
+====
```

Comment:
Please make sure you write proper documentation.. The chain of symbols under a headline has to be _exactly_ as long as the headline in RestructuredText. This also goes for the other file.